t/10-call_command_absolute_path.t

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env perl -w
+
+use strict;
+use warnings;
+use utf8;
+
+use Test::More tests => 1;
+use English qw( -no_match_vars );
+use Bat::Interpreter;
+use Path::Tiny;
+
+use Data::Dumper;
+
+my $interpreter = Bat::Interpreter->new;
+
+my $cmd_file = $PROGRAM_NAME;
+$cmd_file =~ s/\.t/\.cmd/;
+
+local $ENV{"CWD"} = Path::Tiny::path($0)->parent()->absolute()->canonpath();

```

Comment:
This test would only force the issue on a windows execution, maybe instead use a filename that includes de `:` simbol and we can test it on linux too